applications/maplab-node/src/maplab-node.cc

```diff
@@ -43,6 +43,8 @@ DEFINE_int64(
     "Enable visualization of the visual localization observations and inliers "
     "as ROS message.");
 
+DECLARE_double(image_resize_factor);
```

Comment:
Should we give it a more clear name? like for rovioli and maplab independently?

Reply:
as you have `rovioli_image_resize_factor` later, rename this to `maplab_image_resize_factor`

Reply:
Sadly this seems to be our convention at the moment if you look around at the other flags...